packages/typo3-docs-theme/assets/js/menu-expandable.js

```diff
@@ -41,7 +41,7 @@
 
 
 document.addEventListener('DOMContentLoaded', () => {
-  const currentFirstLevelEntries = document.querySelectorAll('li.toctree-l1.active');
+  const currentFirstLevelEntries = document.querySelectorAll('.menu-level-1 > li.active');
```

Comment:
Do we need the "li" here in the selector? Querying the tag and the class looks too much?

Reply:
I cannot see any disadvantage in being specific? We querried for li before as well. And also I believe this part of the JS will soon be gone as we are able now to remove the unwanted levels of menus from the HTML output instead of hiding them by JS